### PR TITLE
Change proto to use string to represent type info

### PR DESCRIPTION
**Pull request by yuzhu**

### CODE REVIEW

**Comment by yuzhu** [comment]
@ZacBlanco @calvinjia  change the way we represent types by simply using string to represent column types.

File: experimental/table/server/under/hive/src/main/java/alluxio/table/under/hive/HiveUtils.java [hunk 1]
```diff
@@ -141,107 +139,92 @@ public static Storage toProto(StorageDescriptor sd, PathTranslator translator)
...
-        if (data != null) {
-          builder.setData(ColumnStatisticsData.newBuilder()
-              .setBooleanStats(BooleanColumnStatsData.newBuilder()
-                  .setNumTrues(data.getNumTrues()).setNumFalses(data.getNumFalses())
-                  .setNumNulls(data.getNumNulls()).setBitVectors(data.getBitVectors())
-                  .build()).build());
-        }
-        break;
+    builder.setColName(colStats.getColName()).setColType(colStats.getColType());
+    if (colStats.getStatsData().isSetBooleanStats()) {
```
Comment (gpang) [inline experimental/table/server/under/hive/src/main/java/alluxio/table/under/hive/HiveUtils.java]: NIT: `colStats.getStatsData()` is used many times, can we just save it to a local variable?

Comment (yuzhu) [reply experimental/table/server/under/hive/src/main/java/alluxio/table/under/hive/HiveUtils.java]: fixed

**Comment by gpang** [comment]
alluxio-bot, merge this please

**Pull request merged**